.github/workflows/test.yml

```diff
@@ -10,84 +10,60 @@ jobs:
+            CASSANDRA_VERSION: 2.1
+            POSTGRES_VERSION: 9
+          - NODE_VERSION: 12.x
+            MYSQL_VERSION: 5.7
+            MARIADB_VERSION: 10.1
+            CASSANDRA_VERSION: 3
+            POSTGRES_VERSION: 9
+          - NODE_VERSION: 14.x
+            MYSQL_VERSION: 8
+            MARIADB_VERSION: 10.5
+            CASSANDRA_VERSION: 3
+            POSTGRES_VERSION: 13
 
     steps:
     - uses: actions/checkout@v2
 
     # ------------------------------
-    # -- Setup environment for tests
+    # -- Setup node dependencies for tests
     # ------------------------------
-    - name: Create MySQL Container
-      run: docker run --rm -d --name mysql -e "MYSQL_ROOT_PASSWORD=Password12!" -e "MYSQL_DATABASE=sqlectron" -p 3306:3306 -v ${GITHUB_WORKSPACE}/spec/databases/mysql/schema:/docker-entrypoint-initdb.d mysql:${{ matrix.mysql-version }}
-
-    - name: Create MariaDB Container
-      run: docker run --rm -d --name mariadb -e "MYSQL_ROOT_PASSWORD=Password12!" -e "MYSQL_DATABASE=sqlectron" -p 3307:3306 -v ${GITHUB_WORKSPACE}/spec/databases/mysql/schema:/docker-entrypoint-initdb.d mariadb:${{ matrix.maraidb-version }}
-
-    - name: Create Postgres Container
-      run: docker run --rm -d --name postgres -e "POSTGRES_PASSWORD=Password12!" -e "POSTGRES_DB=sqlectron" -p 5432:5432 -v ${GITHUB_WORKSPACE}/spec/databases/postgresql/schema:/docker-entrypoint-initdb.d postgres:${{ matrix.postgres-version }}
-
-    - name: Create Redshift Container
-      run: docker run --rm -d --name redshift -e "POSTGRES_PASSWORD=Password12!" -e "POSTGRES_DATABASE=sqlectron" -p 5433:5432 -v ${GITHUB_WORKSPACE}/spec/databases/redshift/schema:/docker-entrypoint-initdb.d foundryai/postgres8:${{ env.redshift-version }}
-
-    - name: Create Cassandra Container
-      run: docker run --rm -d --name cassandra -p 9042:9042 -v ${GITHUB_WORKSPACE}/spec/databases/cassandra/schema:/docker-entrypoint-initdb.d  cassandra:${{ matrix.cassandra-version }}
-
-    - name: Create SQLServer Container
-      run: docker run --rm -d --name sqlserver -p 1433:1433 -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Password12!" -e "MSSQL_COLLATION=${{ matrix.sqlserver-collation }}" -v ${GITHUB_WORKSPACE}/spec/databases/sqlserver/schema:/docker-entrypoint-initdb.d mcr.microsoft.com/mssql/server:2017-latest
-
-    - name: Create OpenSSH-Server Container
-      run: docker run --rm -d --name openssh-server -p 2222:2222 -e "DOCKER_MODS=linuxserver/mods:openssh-server-ssh-tunnel" -e "PASSWORD_ACCESS=true" -e "USER_NAME=sqlectron" -e "USER_PASSWORD=password" -e "PUBLIC_KEY_DIR=/ssh_files/pub" -v ${GITHUB_WORKSPACE}/spec/ssh_files:/ssh_files ghcr.io/linuxserver/openssh-server
-
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js ${{ matrix.NODE_VERSION }}
       uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install -g npm
+        node-version: ${{ matrix.NODE_VERSION }}
 
+    - run: npm install -g npm
```

Comment:
I think this is an artifact of when this was being tested on node 6 in earlier versions of this package, and I think we can remove it as `npm ci` exists for all versions of node we are testing against.